packages/xstate-test/src/testUtils.ts

```diff
@@ -0,0 +1,22 @@
+import { StatePlan } from '@xstate/graph';
+import { TestModel } from './TestModel';
+
+const testModel = async (model: TestModel<any, any>) => {
+  for (const plan of model.getPlans()) {
+    await model.testPlan(plan);
+  }
+};
```

Comment:
Let's not include `testModel` as a function - this is the same as `testPlans()` which is more explicit, and `testModel` is (perhaps confusingly) used as a variable for "the test model"

```suggestion
```

Reply:
Let's bikeshed this outside of this PR.